src/sasconf.h

```diff
@@ -14,7 +14,11 @@
 
 #include <sys/param.h>
 
-#define __BIGREGION__ 1
+#if defined(__BIGREGION__) && __BIGREGION__ == 0
+# undef __BIGREGION__
+#else
+# define __BIGREGION__ 1
+#endif
```

Comment:
@tuliom, With this change, by default we will be using the values in #ifdef __BIGREGION__ 
For smaller/older systems , we have to use CFLAGS="-D__BIGREGION__=0" CXXFLAGS="-D__BIGREGION__=0" during configure.